packages/devtools_app/lib/debugger_html_plugin.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// We actually use dart:html here, no shim.
```

Comment:
remove the 
// we actually use dart:html here, no shim.
All uses of html_shim are really using dart:html and throwing exceptions on other platforms.

Reply:
My goal here is to point out that this code and its transitive dependencies are intended to only run as web code, in spite of being part of the flutter app.

When we turn down the html app, we want to switch from html shim to regular dart:html.